packages/core-contracts/contracts/utils/Ownable.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../interfaces/IOwnable.sol";
+
+abstract contract Ownable is IOwnable {
+    modifier onlyOwner() {
+        _onlyOwner();
+        _;
+    }
+
+    function acceptOwnership() external virtual override {}
```

Comment:
All these are implemented in this contract.

When owner is set for example, they call `_setOwner(newOwner)`